webui/react/src/shared/styles/antd.scss

```diff
@@ -120,12 +120,8 @@ html {
     font-size: 16px;
   }
   .ant-input-number-status-error,
-  .ant-input-number-status-error:not(.ant-input-number-disabled):not(
-      .ant-input-number-borderless
-    ).ant-input-number,
-  .ant-input-number-status-error:not(.ant-input-number-disabled):not(
-      .ant-input-number-borderless
-    ).ant-input-number:hover {
+  .ant-input-number-status-error:not(.ant-input-number-disabled):not(.ant-input-number-borderless).ant-input-number,
+  .ant-input-number-status-error:not(.ant-input-number-disabled):not(.ant-input-number-borderless).ant-input-number:hover {
     background-color: inherit;
   }
```

Comment:
the scss changes are because we had prettier updated 2 days ago, it shouldn't be changed in this PR